sdk/python/lib/pulumi/runtime/proto/provider_pb2.pyi

```diff
@@ -49,29 +49,62 @@ class ConstructResponse:
     def __init__(self,
                  urn: Optional[str]=None,
                  state: Optional[Struct]=None,
-                 stateDependencies: Optional[Dict[str,PropertyDependencies]]=None) -> void:
+                 stateDependencies: Optional[Dict[str,PropertyDependencies]]=None) -> None:
         pass
 
     class PropertyDependencies:
         urns: List[str]
 
-        def __init__(self, urns: List[str]) -> void:
+        def __init__(self, urns: List[str]) -> None:
             pass
 
     urn: str
     state: Struct
     stateDependencies: Dict[str,PropertyDependencies]
 
 
+class CallRequest:
```

Comment:
Sorry for this stuff here, still haven't gotten to auto-generate it, it's somewhere in the backlog.